File: 백준/Silver/2563. 색종이/색종이.js
let [N, ...input] = require("fs")
  .readFileSync("dev/stdin")
  .toString()
  .split("\n");
input = input.map((el) => el.split(" "));
const arr = Array.from(Array(100), () => Array(100).fill(0));
let count = 0;

for (let i = 0; i < N; i++) {
  let startX = +input[i][0]; 
  let startY = +input[i][1]; 
  for (let l = 0; l < 10; l++) {
    for (let m = 0; m < 10; m++) {
      if (arr[startX + l][startY + m] !== 0) {
        continue;
      }
      if (arr[startX + l][startY + m] === 0) {
        arr[startX + l][startY + m] = 1;
        count++;
      }
    }
  }
}
console.log(count);

File: 백준/Silver/2563. 색종이/색종이.py
n = int(input())
array = [[0] * 100 for _ in range(100)]

for _ in range(n):
    a, b = map(int,input().split())
    for i in range(a, a + 10):
        for j in range(b, b + 10):
            array[i][j] = 1
            
cnt = 0

for i in range(100):
    cnt += array[i].count(1)

print(cnt)